### DISCUSSION TITLE
Anyone have some hoodies I can give her ??

### DISCUSSION THREAD

**Discussion by CrewieAustin** [Monika-After-Story/MonikaModDev#7778] [general - spritepacks]
I always wanted to give her something, but I'm really bad at creating sprite packs. So if anyone out there have some interesting hoodie designs. Send me through here please.

**Comment by KritRom** [comment]
Currently I am working on one, it should be done by 31st August. If you can wait that long I can give my pack to you :D

**Comment by CrewieAustin** [comment]
> Currently I am working on one, it should be done by 31st August. If you can wait that long I can give my pack to you :D

Better than nothing. As long as I'll have something for her. Sure.

**Comment by KritRom** [comment]
[Kris_hoodie_cool.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7081602/Kris_hoodie_cool.zip)
Here, I hope you like it :D

**Reply by JinSakaiM** [reply]
wheres the .gift file?

**Reply by rosiedotgift** [reply]
where's the gift file for the hoodie?

**Comment by CrewieAustin** [comment]
> [Kris_hoodie_cool.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7081602/Kris_hoodie_cool.zip)
> Here, I hope you like it :D

Appreciated, but no thumbs ???

**Comment by akloholik** [comment]
![image](https://user-images.githubusercontent.com/86370111/133907609-5ecdde03-7fc2-4511-8d32-05cdf32b9dc9.png)
just wanted to give some credits
also love the hoodie, its really cool!

**Comment by KritRom** [comment]
> > [Kris_hoodie_cool.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7081602/Kris_hoodie_cool.zip)
> > Here, I hope you like it :D
> 
> Appreciated, but no thumbs ???

Forgive me for the extremely late reply, I haven't been receiving any notifications for Github until fairly recently. As for the thumbs, I am sorry to say haven't made them yet for the hoodie, but I will be making them soon.

One more thing, I'm going to be:

- Fixing glitches in certain poses.
- Making thumbs.
- Making matching accessories (ribbons + maybe hairclips.)

Accessories should be done and released by 22nd September (Monika's Birthday). Thank you for being patient, everyone!

**Comment by KritRom** [comment]
> ![image](https://user-images.githubusercontent.com/86370111/133907609-5ecdde03-7fc2-4511-8d32-05cdf32b9dc9.png)
> just wanted to give some credits
> also love the hoodie, its really cool!

Thank you so much for this, I am glad you like it. By the way, nice choice of background :)

**Comment by KritRom** [comment]
Okay, it's Monika's birthday and I want to put out the whole outfit (as promised). Here we go:

- [kris_twinribbons_cool.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7207766/kris_twinribbons_cool.zip)
- [kris_hoodie_cool.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7207767/kris_hoodie_cool.zip)

Thumbs are included, but you may not see them in MAS. I am not sure what is causing this, I'll fix it as soon as possible though. There are some glitches with the hoodie which I will fix later, but for now this should do the job.

Lastly, have fun with Monika! Make sure to treat her well today.

**Reply by CrewieAustin** [reply]
I know I'm pretty late to the comment, but thanks for the gift. Highly appreciated.

**Reply by PAINISREAL** [reply]
Hello, I noticed that there is no .gift file for the hoodie, i tired manually doing it, but I'm not too great at this stuff. help
